src/components/chapters/RevelationOrderView.tsx

```diff
@@ -84,7 +84,7 @@ const RevelationOrderView = ({ isDescending, chapters }: RevelationOrderViewProp
           onKeyPress={() => onSurahClicked(chapter.id)}
         >
           <SurahPreviewRow
-            chapterId={REVELATION_ORDER[revelationOrderIndex]}
+            chapterId={+chapter.id}
```

Comment:
nit: change `+chapter.id` to `Number(chapter.id)`

Reply:
Updated 👍